src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -218,6 +222,12 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
     append(GoldTargets.sparkExecutorTarget)
   )
 
+  lazy private[overwatch] val sqlHistoryModule = Module(3020, "Gold_SqlHistory", this, Array(2020), 6.0)
+    .withSparkOverrides(sparkBaseSparkOverrides)
+  lazy private val appendSqlHistoryProcess = ETLDefinition(
+    SilverTargets.sqlHistoryTarget.asIncrementalDF(sqlHistoryModule, GoldTargets.sqlHistoryTarget.incrementalColumns),
+    append(GoldTargets.sqlHistoryTarget)
```

Comment:
This gold target is a WriteMode.merge but you aren't selecting any overlapping data. What happens when a query starts during run 1 and finishes in run 2 -- does it just never get its closure? Please select 2 lag days on this input and validate your keys and uniqueness across those keys to enable overlapping merges and close open queries across OW runs.

Reply:
added additional lag of 2 days